db/seeds.d/106-mail_notifications.rb

```diff
@@ -31,6 +31,13 @@
      :mailer => 'Katello::ErrataMailer',
      :method => 'promote_errata',
      :subscription_type => 'alert'
+    },
+
+    {:name => :subscription_expiry,
+     :description => N_('A list of Subscriptions expiring in 30 days'),
+     :mailer => 'Katello::SubscriptionMailer',
+     :method => 'subscription_expiry',
+     :subscription_type => 'report'
```

Comment:
This `:subscription_type` value doesn't seem to be used anywhere (for any of the other mailers, either), so this may not matter much.  But I would say this is more of an "alert" than a "report," since the email will eventually _link_ to a report anyway.

Reply:
@jeremylenz subscription_type determines the values in the drop down. 'report' as the subscription_type shows - No emails, Daily, Weekly, Monthly. 'alert' as the subscription_type shows - No emails, subscribe.

Reply:
@sseelam2 aah okay, didn't catch that!